docs/concepts/validation_decorator.md

```diff
@@ -392,31 +319,16 @@ print(foo())
 
 ### Validation exception
 
-Currently upon validation failure, a standard Pydantic [`ValidationError`][pydantic_core.ValidationError] is raised.
-See [model error handling](models.md#error-handling) for details.
-
-This is helpful since its `str()` method provides useful details of the error which occurred and methods like
-`.errors()` and `.json()` can be useful when exposing the errors to end users. However, `ValidationError` inherits
-from `ValueError` **not** `TypeError`, which may be unexpected since Python would raise a `TypeError` upon invalid
-or missing arguments. This may be addressed in future by either allowing a custom error or raising a different
-exception by default, or both.
-
-### Coercion and strictness
-
-Pydantic currently leans on the side of trying to coerce types rather than raise an error if a type is wrong,
-see [model data conversion](models.md#data-conversion) and `@validate_call` is no different.
```

Comment:
Removed as this was mentioned in the _#Parameter types_ section, and type coercion is not really specific to validate_call, it's a general Pydantic pattern.

Reply:
Maybe we could link to the data conversion section in the docs above?

I think we should probably leave the validation exception section for folks who might not be super familiar with Pydantic - I think this is helpful context to have.

Reply:
Added a ref to [model data conversion](models.md#data-conversion).

The validation exception wasn't removed but simplified